Mu2eG4/geom/psEnclosure_v05.txt

```diff
@@ -0,0 +1,102 @@
+// The configuration to be used with the as-built design, reflected
+// The configuration to be used with the as-built design, reflected
```

Comment:
Perhaps remove the duplicated line and mention the new note in the docdb?

Reply:
I've updated the comment here, and removed the duplicate.